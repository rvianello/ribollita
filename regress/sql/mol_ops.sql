select mol_to_smiles(mol_kekulize(mol_from_smiles('c1ccccc1')));

select mol_to_smiles(fragment) from mol_fragments(mol_from_smiles('C.CC.COC'));

select mol_formal_charge(mol_from_smiles('CC(=O)[O-]'));


