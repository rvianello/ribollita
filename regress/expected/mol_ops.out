select mol_to_smiles(mol_kekulize(mol_from_smiles('c1ccccc1')));
 mol_to_smiles 
---------------
 C1=CC=CC=C1
(1 row)

select mol_to_smiles(fragment) from mol_fragments(mol_from_smiles('C.CC.COC'));
 mol_to_smiles 
---------------
 C
 CC
 COC
(3 rows)

select mol_formal_charge(mol_from_smiles('CC(=O)[O-]'));
 mol_formal_charge 
-------------------
                -1
(1 row)

